pkg/testutil/testutil.go

```diff
@@ -450,17 +450,6 @@ func (c *Cmd) AssertOutStreamsExactly(stdout, stderr string) {
 	c.AssertOutStreamsWithFunc(fn)
 }
 
-func (c *Cmd) AssertNoOut(s string) {
-	c.Base.T.Helper()
-	fn := func(stdout string) error {
-		if strings.Contains(stdout, s) {
-			return fmt.Errorf("expected not to contain %q, got %q", s, stdout)
-		}
-		return nil
-	}
-	c.AssertOutWithFunc(fn)
-}
-
```

Comment:
Is this change relevant to the bug?
Otherwise please consider splitting the commit